integrations/nifi/nifi-rest/nifi-rest-client/nifi-rest-client-api/src/main/java/com/thinkbiganalytics/nifi/rest/client/AbstractNiFiProcessorsRestClient.java

```diff
@@ -151,7 +157,7 @@ private ProcessorConfigDTO createConfig(String schedStrategy, String schedPeriod
     
     private void delay(long millis) {
         try {
-            Thread.sleep(millis);;
+            Thread.sleep(millis);
         } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)